jenkins-pipelines/artifacts-ubuntu2404-nonroot.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+artifactsPipeline(
+    test_config: 'test-cases/artifacts/ubuntu2204.yaml',
```

Comment:
I think it should be 'test-cases/artifacts/ubuntu2404.yaml' instead of `ubuntu2204.yaml`?
and this `artifacts-ubuntu2404-nonroot.jenkinsfile` file should be located under `nonroot-offline-install` folder?

Reply:
I created PR to trigger this jobs in 2024.1
https://github.com/scylladb/scylla-enterprise-pkg/pull/281

configured these tests manually under releng-testing, running verification

Reply:
@Annamikhlin seems like it wrong also in master, it was missed in the review of the original PR
https://github.com/scylladb/scylla-cluster-tests/pull/8354

we'll backport afterwards

Reply:
change my mind and I did the changes here
since the file layout is different, now aligned it correctly to how 2024.1 is setup

Reply:
@Annamikhlin 

all the jobs failed, but cause of more backports missing
- https://github.com/scylladb/scylladb/issues/18568

but I think this one can be merged

Reply:
I run also verification based on my branch and your branch in releng-testing

artifacts-ubuntu2404-test:
- https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts/job/artifacts-ubuntu2404-test/2/ - passed successfully 

artifacts-ubuntu2404-arm-test:
- irst run passed -  https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts/job/artifacts-ubuntu2404-arm-test/2/ 
- second run failed - https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts/job/artifacts-ubuntu2404-arm-test/4/

Reply:
For offline installer verification - 
please take a look on the failures
- https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts-offline-install/job/artifacts-ubuntu2404-test/3/
- https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts-offline-install/job/artifacts-ubuntu2404-arm-test/3/

- https://jenkins.scylladb.com/job/enterprise-2024.1/job/releng-testing/job/artifacts-offline-install/job/artifacts-ubuntu2404-nonroot-test/3/ - passed successfully -